client/layout/masterbar/style.scss

```diff
@@ -3,9 +3,14 @@ $autobar-height: 20px;
 
 // The WordPress.com Masterbar
 .masterbar {
-	background: $masterbar-color;
-	border-bottom: 1px solid darken( $masterbar-color, 4% );
-	color: $white;
+	border-bottom: 1px solid;
+	@include theme-multiple(
+		(
+			'background': --masterbar-bg-color,
+			'border-color': --masterbar-border-color,
+			'color': --masterbar-color,
+		)
+	);
```

Comment:
@folletto I've added a comment to your other question below as to why we'd want to use a mixin when using custom properties (tl;dr: to provide a fallback). 

The idea behind this mixin in particular though was to improve the developer experience by providing a mixin that allows the developer to declare multiple properties at once. The same could, of course, be achieved by using the theme mixin three times:
```
@include theme( background, --masterbar-bg-color );
@include theme( border-color, --masterbar-border-color  );
@include theme( color, --masterbar-color );
```
It does potentially reduce legibility of the code though and thus the potential long term cost might outweight the short term developer experience benefit. Maybe just remove it?